cmd/podman/create.go

```diff
@@ -781,11 +784,10 @@ func parseCreateOpts(ctx context.Context, c *cli.Context, runtime *libpod.Runtim
 		VolumesFrom: c.StringSlice("volumes-from"),
 	}
 
-	if !config.Privileged {
-		if err := parseSecurityOpt(config, c.StringSlice("security-opt")); err != nil {
-			return nil, err
-		}
+	if err := parseSecurityOpt(config, c.StringSlice("security-opt")); err != nil {
```

Comment:
I don't know if this is safe - it might turn apparmor and seccomp on if we're privileged, which we do not want

Reply:
Yes why would we turn off the error reporting here?

Reply:
@mheon right, reworked it so that doesn't happen. @rhatdan the error reporting is still there.